### ISSUE TITLE
Anti-H3-H4 antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6615]
**Preferred term label:** Anti-H3-H4 antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Anti-histone positivity

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
I am not sure if this is a great HPO term -- I am guessing there is an antibody that reacts against both of these antigens so that we are not sure whether it is H3 or H4? It seems like most of the modern antibodies at least claim to be specific for one or another. I would suggest that if we are not sure about the specificity of the antibody the it is better to annotate against the parent term ``Anti-histone antibody positivity``, otherwise, we would also have to create terms for the many other "fuzzy" results and this would be confusing.
Please reopen if this is a bad idea.

**Comment by eswietlik** [comment]
@pnrobinson This antibody (similarly to anti-H2A-H2B Ab) is specific to the dimer as opposed to others that are specific to monomers

**Comment by pnrobinson** [comment]
@eswietlik Can you provide PMIDs for these antibodies? Is it this? https://pubmed.ncbi.nlm.nih.gov/1864977/

**Comment by eswietlik** [comment]
@pnrobinson Thank you, that is correct also PMID: 7986226, PMID: 3874029

**Comment by pnrobinson** [comment]
added term
